src/server/server.config.js

```diff
@@ -148,6 +148,32 @@ const conf = convict({
       }
     }
   },
+  mongodb: {
+    uri: {
+      doc: 'Mongo DB URI',
+      format: '*',
+      env: 'MONGO_DB_URI',
+      default: ''
+    },
+    user: {
```

Comment:
uri should be enough it can include everything

Reply:
done